.gitignore

```diff
@@ -120,3 +120,8 @@ src/**/.deps
 src/**/.dirstamp
 mac/ccextractorGUI
 linux/ccextractorGUI
+linux/CMakeFiles/
+linux/CMakeCache.txt
+linux/cmake_install.cmake
```

Comment:
If you are adding CMake Ignore files you should probably add for mac too.

Reply:
Isn't it better to Add it as a separate PR?

Reply:
It is depend on him to whether add them in this PR or make a new PR